<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SonarPhony</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="waterfall_t" name="waterfall">
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="columnPulser_t" name="columnPulser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>48</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Debug</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="output">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="toolTip">
        <string>Begin running the sonar.</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="toolTip">
        <string>Stop the sonar (may take a few seconds).</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minRangeSB">
       <property name="toolTip">
        <string>Minimum depth range</string>
       </property>
       <property name="suffix">
        <string> ft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxRangeSB">
       <property name="toolTip">
        <string>Maximum depth range</string>
       </property>
       <property name="suffix">
        <string> ft</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>270</number>
       </property>
       <property name="value">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dataDockWidget">
   <property name="windowTitle">
    <string>Data</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Depth to sea floor from transducer head.</string>
       </property>
       <property name="text">
        <string>Depth</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="depth">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ft</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="maxRange">
       <property name="text">
        <string>-- ft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="minRange">
       <property name="text">
        <string>-- ft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>Water temperature.</string>
       </property>
       <property name="text">
        <string>Temperature</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QProgressBar" name="batteryCharge">
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%p %</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="batteryVoltage">
       <property name="toolTip">
        <string>Battery voltage.</string>
       </property>
       <property name="text">
        <string>-- V</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Maximum range on latest ping.</string>
       </property>
       <property name="text">
        <string>Max Range</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Battery life remaining.</string>
       </property>
       <property name="text">
        <string>Battery</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="temperature">
       <property name="text">
        <string>-- C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="gain">
       <property name="text">
        <string>-- %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Minimum range on latest ping.</string>
       </property>
       <property name="text">
        <string>Min Range</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="toolTip">
        <string>Sonar gain.</string>
       </property>
       <property name="text">
        <string>Gain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>---- -- -- --:--:--.---</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="text">
    <string>Background</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="iconText">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Modify settings of this program.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About this software.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>waterfall_t</class>
   <extends>QLabel</extends>
   <header>waterfall.hh</header>
  </customwidget>
  <customwidget>
   <class>columnPulser_t</class>
   <extends>QLabel</extends>
   <header>columnPulser.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
